src/content/en/fundamentals/native-hardware/capturing-images/index.md

```diff
@@ -2,116 +2,242 @@ project_path: /web/_project.yaml
-experience, or it could be delegated to another app on the user's device.
+Many browsers now have the ability to access video and audio input from the
+user. However, depending on the browser it might be a full dynamic and inline
+experience, or it could be delegated to another app on the user's device. On top
+of that, not every device even has a camera. So how can you create an experience
+that uses a user generated image that works well everywhere?
 
 ## Start simple and progressively
 
 The easiest thing to do is simply ask the user for a pre-recorded
-file. Do this by creating a simple file input element and adding 
-an `accept` filter that indicates we can only accept image files and ideally we 
-will get them directly from the camera.
+file. There are a lot of good options here to give you a great progressive enhancement story.
 
-    <input type="file" accept="image/*" capture>
+### Ask for a URL
+This is the best supported but least satisfying option - get the user to give you a URL, and then
+use that. For just displaying the image this works everywhere - create an `img` element, set the
+`src` and you're done.
+
+It's more complicated than that, though, if you want to manipulate the image in any way. CORS will
+prevent you from accessing the actual pixels unless the server sets the appropriate headers, and the
+only practical way around that is to run a proxy server.
+
+### File input
+You can also use a simple file input element, including an `accept` filter that indicates we only
+want image files.
+
+    <input type="file" accept="image/*">
 
-This method works on all platforms. On desktop it will prompt the user to 
-upload an image file from the file system. In Safari
-on iOS this method will open up the camera app, allowing you to capture an image 
-and then send it back to the web page; on Android this method will give the user 
-a choice of which app to use to capture the image before sending it back to the
-web page.
+This method works on all platforms. On desktop it will prompt the user to
+upload an image file from the file system. In Chrome and Safari on iOS and Android this method will
+give the user a choice of which app to use to capture the image, including the option of taking a
+photo directly with the camera or choosing an existing image file.
 
-The data can then be attached to a `<form>` or manipulated with JavaScript by 
-listening for an `onchange` event on the input element and then reading 
+<img class="attempt-right" src="images/ios-chooser.png" />
+<img class="attempt-right" src="images/android-chooser.png" />
+
+The data can then be attached to a `<form>` or manipulated with JavaScript by
+listening for an `onchange` event on the input element and then reading
 the `files` property of the event `target`.
 
-### Capture a single frame
+    <input type="file" accept="image/*" id="camera">
+    <script>
+      const camera = document.getElementById('camera');
```

Comment:
I would use something different from `camera` as this is not necessarily the camera unless developer sets the `capture` attribute.